Build fails on updatedb.cpp. The upstream uses `PRIu64' without including
inttypes.h header file. Adding it here.

Signed-off-by: leedagee <leedageea@gmail.com>
--- plocate-1.1.18/dprintf.h	2023-01-29 04:45:19.000000000 +0800
+++ plocate-1.1.18-headers/dprintf.h	2023-05-24 12:59:38.479728079 +0800
@@ -1,6 +1,7 @@
 #ifndef _DPRINTF_H
 #define _DPRINTF_H 1
 
+#include <inttypes.h>
 #include <stdio.h>
 
 extern bool use_debug;
